addons/calendar/wizard/calendar_alarm_test.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from datetime import datetime
+
+from odoo import fields, models, tools
+
+
+class TestCalendarAlarm(models.TransientModel):
+    _name = 'calendar.alarm.test'
+    _description = 'Test Reminder Wizard'
+
+    event_id = fields.Many2one('calendar.event', string='Event', required=True, ondelete='cascade')
+
+    alarm_id = fields.Many2one('calendar.alarm', string='Reminder', required=True, ondelete='cascade')
+
+    def send_reminder_test(self):
+        self.ensure_one()
+        ctx = dict(self.env.context)
+        self = self.with_context(ctx)
+
+        if self.alarm_id.alarm_type == 'email':
+            for attendee in self.event_id.attendee_ids:
+                self.alarm_id.mail_template_id.send_mail(attendee.id, force_send=True, notif_layout='mail.mail_notification_light')
```

Comment:
Mhh.
This will send an email to the attendees of the event.
I think this task is intended so we can test the email and send it to ourselves.
This kind of defeats the purpose.

Maybe we should put the original "mailing.mailing.test" into the mail module, rename it "notification.test" and allow it to be generic.
It would be overridden by both mass_mailing and calendar to provide email/bus/sms testing capabilities.
Maybe using a "mail_test_target" that is a selection between "mailing_mailing" and "calendar_alarm".
Then the method to send the notification would be based on the mail_test_target.
And we could use a "notification_type" that would be a selection between "mail" and "push", that gets "sms" added by the sms module?

Or maybe I'm crazy and this is over-engineerd?
I would like @tde-banana-odoo opinion on this.

Reply:
For now, I've just modify this to send an email to the actual user.